### ISSUE TITLE
Too noise logging about telemetry

### ISSUE THREAD

**Issue by mik-laj** [googleapis/python-bigquery#441]
Hello,

In the Apache Airflow project, we use the BigQuery library, but recently we've started to see annoying log message when the library is loaded. It is enough that the library is loaded and there is an message every time. 

In my opinion, this message should be of a lower level (DEBUG) so that it is not displayed much less often or is displayed only when the client is initialized. 
```
import logging
logging.basicConfig(level=logging.INFO)

from google.cloud import bigquery
```
Output: 
```
INFO:google.cloud.bigquery.opentelemetry_tracing:This service is instrumented using OpenTelemetry. OpenTelemetry could not be imported; please add opentelemetry-api and opentelemetry-instrumentation packages in order to get BigQuery Tracing data.
```

Related issue: https://github.com/apache/airflow/issues/13131

CC: @tswast

**Comment by mik-laj** [comment]
@tswast this problem is in part related to the implementation of your AIP. https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-8+Split+Providers+into+Separate+Packages+for+Airflow+2.0
Now we need to load a few more classes to be able to read the attributes of some classes, which unfortunately leads to the BigQuery client being loaded as well.

**Comment by tswast** [comment]
Sent https://github.com/googleapis/python-bigquery/pull/442 to drop the logging level to DEBUG

**Comment by tswast** [comment]
Reading the Airflow issue, I think the problem is that there is any sort of message at *import* time. I'll see if there's a different way we can show this warning when opentelemetry is specifically requested. Or perhaps just remove it.

**Comment by tswast** [comment]
This is fixed in `google-cloud-bigquery==2.6.2`.
